src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryCustomObject.cs

```diff
@@ -49,9 +77,14 @@ public string GetString()
                 return _element.GetString();
             }
 
+            public bool GetBoolean()
+            {
+                return _element.GetBoolean();
+            }
+
             public object ToObject()
             {
-                return _element.GetRawText();
+                return _element;
```

Comment:
>return _element; [](http://example.com/codeflow?start=16&length=16)

I don't think there's any correct way to implement the ToObject() method. 
Can we just remove it entirely from the interface? 
In theory, if we really had to create an .net Object for it, an adatper could provide it leveraging the existing type support.

Reply:
Removed, we just return the Impl now